app/views/approvedApplication/approved.scala.html

```diff
@@ -1,55 +1,80 @@
-@()(implicit request: Request[_], userFullName: Option[String])
-
+@import model.ApprovedApplication
+@import org.joda.time.format.DateTimeFormat
+@(application: ApprovedApplication)(implicit request: Request[_], userFullName: Option[String])
 
 @main(title = "API Gatekeeper Dashboard",userFullName = userFullName ) {
-<div class="content__body">
-
-    <nav class="breadcrumb-nav">
-        <ul>
-            <li class="breadcrumb-nav__item"><a href="dashboard">Application names</a></li>
-            <li class="breadcrumb-nav__item breadcrumb-nav__item--trail" aria-hidden="true"></li>
-            <li class="breadcrumb-nav__item">application name</li>
-        </ul>
-    </nav>
-
-    <header>
-        <h1 class="flush--top">Application name here</h1>
-    </header>
-
-    <details>
-        <summary class="status--verified">Verified</summary>
-        <div>
-            <p class="panel-indent">The submitter has verified that they are still associated with the email address associated with this application.</p>
+
+    @defining(DateTimeFormat.forPattern("dd MMMM yyyy")) { dateFormatter =>
+        <div class="content__body">
+
+            <nav class="breadcrumb-nav">
+                <ul>
+                    <li class="breadcrumb-nav__item"><a href="dashboard">Application names</a></li>
```

Comment:
The requirement is that the text should actually be 'Application names'. Leaving it as is.